crypto/tls/tls.go

```diff
@@ -85,3 +114,31 @@ func (cfg *ClientConfig) GetGRPCDialOptions(enabled bool) ([]grpc.DialOption, er
 
 	return []grpc.DialOption{grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig))}, nil
 }
+
+func mapCipherNameToID(cipherSuiteNames []string) ([]uint16, error) {
```

Comment:
[Nit] I just think it's more helpful to the reader if the function name reflects the inputs and outputs being plural :)
```suggestion
func mapCipherNamesToIDs(cipherSuiteNames []string) ([]uint16, error) {
```